src/algebra/algebra/basic.lean

```diff
@@ -144,19 +144,16 @@ end
 
 @[priority 200] -- see Note [lower instance priority]
 instance to_module : module R A :=
-{ one_smul := by simp [smul_def''],
-  mul_smul := by simp [smul_def'', mul_assoc],
-  smul_add := by simp [smul_def'', mul_add],
-  smul_zero := by simp [smul_def''],
-  add_smul := by simp [smul_def'', add_mul],
-  zero_smul := by simp [smul_def''] }
+{ one_smul := by simp [smul_def],
+  mul_smul := by simp [smul_def, mul_assoc],
+  smul_add := by simp [smul_def, mul_add],
+  smul_zero := by simp [smul_def],
+  add_smul := by simp [smul_def, add_mul],
+  zero_smul := by simp [smul_def] }
 
 -- from now on, we don't want to use the following instance anymore
 attribute [instance, priority 0] algebra.to_has_scalar
 
-lemma smul_def (r : R) (x : A) : r • x = algebra_map R A r * x :=
```

Comment:
I think it might be important that this appears after `attribute [instance, priority 0] algebra.to_has_scalar`, as it affects which instance appears in the statement.

Perhaps the solution is just to make the double-primed version private?